src/app/modules/semaforo-priorizacion/components/semaforo-priorizacion.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { SemaforoService } from '../service/semaforo.service';
+
+
+@Component({
+    selector: 'semaforo-priorizacion',
+    templateUrl: './semaforo-priorizacion.html',
+    styleUrls: ['./scss/semaforo.scss']
+
+})
+
+export class SemaforoPriorizacionComponent implements OnInit {
+    @Output() onChangeEvent = new EventEmitter<any>();
+    @Input() name;
+    @Input() prioridad;
+    opciones;
+
+    constructor(
+        private semaforoService: SemaforoService) { }
+
+    ngOnInit(): void {
+        this.semaforoService.findByName('com').subscribe(res =>  this.opciones = res.options);
```

Comment:
Se pasa un `name` por input pero después está 'com' harcodeado. 
```suggestion
        this.semaforoService.findByName(name).subscribe(res =>  this.opciones = res.options);
```

Reply:
Buen catch! Updeteado!